.github/workflows/upload_all.yml

```diff
@@ -141,4 +141,9 @@ jobs:
         run: make upload_ipa
       - name: Publish
         if: github.event.inputs.draft != 'true'
-        run: echo ::notice::Publishing on the App Store not yet supported, please do it manually.
+        working-directory: platform/ios
+        env:
+          POLYPOD_APPLE_IOS_APPSTORE_API_ISSUER_ID: ${{ secrets.POLYPOD_APPLE_IOS_APPSTORE_P12_FILE_BASE64 }}
+          POLYPOD_APPLE_IOS_APPSTORE_API_PRIVATE_KEY_BASE64: ${{ secrets.POLYPOD_APPLE_IOS_APPSTORE_API_PRIVATE_KEY_BASE64 }}
+          POLYPOD_APPLE_IOS_APPSTORE_API_KEY_ID: ${{ secrets.POLYPOD_APPLE_IOS_APPSTORE_API_KEY_ID }}
+        run: make distribute_app_to_beta_testers
```

Comment:
Naming nit: Not exactly _beta_ what we're publishing here, we normally call it _internal testing_.

Reply:
Ok, I will update to something like `release_app_to_external_testers`. Note this command does exactly this and nothing more, it does not release the app to the AppStore - this will be done in next PRs.